lib/daru/dataframe.rb

```diff
@@ -2006,6 +2005,18 @@ def split_by_category cat_name
 
     private
 
+    def headers
+      if index.is_a?(MultiIndex)
+        Daru::Index.new(index.name) | @vectors
+      else
+        Daru::Index.new([index.name]) | @vectors
+      end
+    end
+
+    def row_headers
```

Comment:
Just a replacement of [this line](https://github.com/SciRuby/daru/pull/323/files#diff-cfd587da29af6a7a2d1f98c614ef123fL1905)